bindings/ruby/Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    regorusrb (0.1.0)
+    regorusrb (0.1.5)
```

Comment:
@anakrish I think this is what needed to be bumped? `Gemfile.lock` is a generated file, I'm updating the `bindings/ruby/README.md` with instructions on how to generate and update in the future.